.github/CONTRIBUTING.md

```diff
@@ -2,24 +2,54 @@
 
 :tada: Thanks, for considering to contribute to EKO!
 
-:pray: For the sake of simplicity we switch below to imperative language, however, please read a "Please" infront of everything.
-
-- :brain: Be reasonable and use common sense when contributing: we added some points we would like to highlight below
-- :family: Follow our [Code of Conduct](https://github.com/N3PDF/eko/blob/master/.github/CODE_OF_CONDUCT.md) and use the provided
-  [Issue Templates](https://github.com/N3PDF/eko/issues/new/choose)
+:pray: For the sake of simplicity we switch below to imperative
+language, however, please read a "Please" infront of everything.
+
+-   :brain: Be reasonable and use common sense when contributing: we
+    added some points we would like to highlight below
+-   :family: Follow our [Code of
+    Conduct](https://github.com/N3PDF/eko/blob/master/.github/CODE_OF_CONDUCT.md)
+    and use the provided [Issue
+    Templates](https://github.com/N3PDF/eko/issues/new/choose)
+
+## Tools
+
+-   [`poetry`](https://github.com/python-poetry/poetry) is the
+    dependency manager and packaging back-end of choice for this
+    project, refers to official [installation
+    guide](https://python-poetry.org/docs/#installation)
```

Comment:
I would link their git repo https://github.com/python-poetry/poetry, since some instructions on their docs are not up to date.

Reply:
The repo is actually linked (line 17). But referring to repo for installation instructions is risky:
- repo by default is `master`, and so even instructions might change (they are unstable) and refer to the latest release (in the best case)
- website by default is `stable`, so it's the advised version to be used

In general, you expect to find official instructions on official website, and developer instructions on repo (I'd say).